ip-docker/Dockerfile

```diff
@@ -1,23 +1,43 @@
-# FROM python:3.9-slim-buster
-# WORKDIR /ip_service
-# COPY requirements.txt requirements.txt
-# # cv2 dependencies
-# RUN apt-get update && apt-get install ffmpeg libsm6 libxext6  -y
-
-# RUN pip install -r requirements.txt
-# COPY . .
-# CMD ["flask", "run", "--host", "0.0.0.0", "--debugger"]
-
 # FOR ARM64
 FROM --platform=linux/amd64 python:3.9-slim-buster
-EXPOSE 5000
+
+# Set working directory
 WORKDIR /ip_service
-COPY requirements.txt requirements.txt
-# cv2 dependencies
-RUN apt-get update && apt-get install ffmpeg libsm6 libxext6  -y
-RUN apt-get install -y binutils libproj-dev gdal-bin ffmpeg libgdal-dev
-RUN export CPLUS_INCLUDE_PATH=/usr/include/gdal
-RUN export C_INCLUDE_PATH=/usr/include/gdal
-RUN pip install -r requirements.txt
-COPY . .
+
+# Expose the port the app runs on
+EXPOSE 8080
+
+# Install system dependencies required for Poetry and your project
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       ffmpeg \
+       libsm6 \
+       libxext6 \
+       binutils \
+       libproj-dev \
+       gdal-bin \
+       libgdal-dev \
+       curl
+
+# Set environment variables for GDAL
+ENV CPLUS_INCLUDE_PATH=/usr/include/gdal \
+    C_INCLUDE_PATH=/usr/include/gdal
+
+# Install Poetry
+RUN pip install poetry
+
+# Ensure that Poetry is in the PATH
+ENV PATH="${PATH}:/root/.poetry/bin"
```

Comment:
This smells suspicious - why do you need it?

Reply:
honestly did not need this, removed it from Dockerfile